src/index.html

```diff
@@ -11,8 +11,65 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div>
+        <img
+          class="card_image"
+          src="./images/imac.jpeg"
+          alt="imac-product"
+        />
+      </div>
+
+      <div>
+        <h2 class="card_product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div>
+        <p class="card_product-code">Product code: 195434</p>
+      </div>
+
+      <div class="stars">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <p class="stars_reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card_price-conteiner">
+        <p class="card_price">Price:</p>
+        <p class="card_price-number">$2,199</p>
+      </div>
+
+      <div>
+        <button
```

Comment:
The class name 'card_button' should be 'card__button' according to BEM naming conventions.